scripts/BuildLumberMill/BuildLumberMill.gml

```diff
@@ -0,0 +1 @@
+with(oBuilder){ building = oLumberMill; }
```

Comment:
Could move this code into a separate script: `Build(building)`, then we wouldn't have to repeat this code in the `BuilderLumberMill`, `BuildQuarry` and `BuildVillage` scripts.